neofetch

```diff
@@ -1585,20 +1596,36 @@ get_de() {
         *xfce5*) de=Xfce5 ;;
         *xfce*)  de=Xfce ;;
         *mate*)  de=MATE ;;
-
-        *MUFFIN*|Cinnamon)
-            de=$(cinnamon --version)
-            de=${de:-Cinnamon}
-        ;;
-
-        *GNOME*)
-            de=$(gnome-shell --version)
-            de=${de/Shell }
-        ;;
+        *GNOME*) de=GNOME ;;
+        *MUFFIN*|Cinnamon) de=Cinnamon ;;
```

Comment:
This could now just be: `*MUFFIN*) de=Cinnamon ;;` as `de` will equal Cinnamon anyway.